SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -473,6 +444,24 @@ bool FLocalDeploymentManager::StartLocalDeploymentShutDown()
 	return true;
 }
 
+bool FLocalDeploymentManager::WaitForGracefulShutdown()
+{
+#if PLATFORM_WINDOWS
+	WindowsRequestGracefulRuntimeShutdown();
+#else
+	NonWindowsRequestGracefulRuntimeShutdown();
+#endif
+	bool bGracefulShutdownSuccess = WaitForRuntimeProcessToShutDown();
+	return bGracefulShutdownSuccess;
+}
+
+bool FLocalDeploymentManager::WaitForForcefulShutdown()
+{
+	RuntimeProcess->Stop();
+	const bool bForcefulShutdownSuccess = WaitForRuntimeProcessToShutDown();
```

Comment:
Nit: Think we can remove this line and just return WaitForRuntimeProcessToShutDown.
I think the reasons to have is as a separate line would be either to cache the value, readability or debuggability. I don't think those are aided in this case.

Reply:
Hmm, I included this as I felt it wasn't super clear what WaitForRuntimeProcessToShutDown was returning. do you think the tradeoff of a slightly less clear function is outweighed by a shorter function?

Reply:
On second read, I think you are right and it does help readability!